<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="point1">
     <item>
      <widget class="QSpinBox" name="X1">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Y1">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Z1">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="point2">
     <item>
      <widget class="QSpinBox" name="X2">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Y2">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Z2">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="point3">
     <item>
      <widget class="QSpinBox" name="X3">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Y3">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Z3">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="point4">
     <item>
      <widget class="QSpinBox" name="X4">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Y4">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Z4">
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>31</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="rotateOX">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот OX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateOY">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>610</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот OY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawSurface">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>680</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>680</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол поворота</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>715</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Точность отрисовки</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>680</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="accuracy">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>710</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.050000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
